tests/arq/src/test/java/org/jboss/test/arquillian/ce/openjdk/OpenJDKMysqlPersistentTest.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.test.arquillian.ce.openjdk;
+
+import org.arquillian.cube.openshift.api.OpenShiftResource;
+import org.arquillian.cube.openshift.api.Template;
+import org.arquillian.cube.openshift.api.TemplateParameter;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+@Template(url = "file://${user.dir}/src/test/resources/openjdk18-mysql-persistent-s2i.json",
+        parameters = {
+                @TemplateParameter(name = "IMAGE_STREAM_NAMESPACE", value = "${kubernetes.namespace:openshift}")
```

Comment:
is openshift good default here? if the xpaas images are deployed on the cluster test will run against released image. Isn't it better to fail if non-default image is not available?

Reply:
This is the default that is added to the template itself: https://github.com/jboss-openshift/application-templates/blob/225f292c72603c021a650daa74b3e78d76a24c22/openjdk/openjdk18-web-basic-s2i.json#L79

Reply:
I know.. but we probably want test to fail instead of testing wrong image and pass.

Reply:
Default will not be used since `kubernetes.namespace` is set always. I can probably remove the default.